beautifultable/beautifultable.py

```diff
@@ -1213,3 +1216,47 @@ def from_csv(self, file_name, header=True, **kwargs):
             for row in csv_reader:
                 self.rows.append(row)
             return self
+
+    def to_df(self):
+        """Export table to dataframe.
+
+        Parameters
+        ----------
+        table : Beautifuletable
+
+        Raises
+        ------
+        AttributeError
+            If header is not there in table
+        """
+
+        headers = self.columns.header
+        headers = list(headers)
+        if not headers:
+            raise AttributeError("Dataframe needs a header")
+
+        # index will act as a row header for beautifultable
+        index = list(self.rows.header)
+        mk_dict = {header: self.columns[header] for header in headers}
```

Comment:
this will lead to unnecessary instantiations. why not use the `data` attribute of the table to get the list of rows. and then use that to generate a dataframe.

Reply:
Done